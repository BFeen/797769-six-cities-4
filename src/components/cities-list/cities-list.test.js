import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list.jsx";


const cityMock = {
  name: `Paris`,
  coordinates: [48.85, 2.34],
};

describe(`CitiesList snapshot testing`, () => {
  it(`CitiesList rendering`, () => {
    const tree = renderer.create(
        <CitiesList
          currentCity={cityMock}
          onItemClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
