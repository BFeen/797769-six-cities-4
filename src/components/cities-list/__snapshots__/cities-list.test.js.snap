// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList snapshot testing CitiesList rendering 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Paris
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Cologne
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Brussels
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Hamburg
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
    >
      <span>
        Dusseldorf
      </span>
    </a>
  </li>
</ul>
`;
